android/src/main/java/com/revenuecat/purchases/hybridcommon/common.kt

```diff
@@ -236,6 +235,21 @@ fun setLogHandler(callback: (logDetails: Map<String, String>) -> Unit) {
     Purchases.logHandler = LogHandlerWithMapping(callback)
 }
 
+/**
+ * Sets a log handler and forwards all logs to completion function. Accepts an OnResult so it can be used from
+ * SDKs that don't have Kotlin configured, since they would error because Kotlin's Function1 would not be found.
```

Comment:
I guess enabling kotlin in Unity will be difficult right? 

So moving forward we should always try to use this Java interface in PHC instead of using lambdas correct? To avoid having duplicate methods.

Reply:
Yes. I am actually not even sure how to enable it since we don't really have a build.gradle, I don't think it's straightforward. This was my bad I shouldn't have created a lambda parameter first place but didn't want to break the other hybrids. I guess lesson learned is do not make a lambda parameter in phc until we migrate everything to Kotlin

Reply:
Actually enabling Kotlin would mean adding a Kotlin file instead or Java. Maybe we can create a dummy Kotlin file and that would take care of it? I'll try that because I would prefer to use the regular lambda version